pyaerocom/helpers.py

```diff
@@ -208,8 +207,8 @@ def tuple_list_to_lists(tuple_list):
 def make_dummy_cube_latlon(
     lat_res_deg: float = 2,
     lon_res_deg: float = 3,
-    lat_range: Optional[Sequence[float]] = None,
-    lon_range: Optional[Sequence[float]] = None,
+    lat_range: list[float] | None = (-90, 90),
+    lon_range: list[float] | None = (-180, 180),
```

Comment:
Now that you have a concrete default, you do not need accept `None`
``` python
    lat_range: list[float] = (-90, 90),
    lon_range: list[float] = (-180, 180),
```

Reply:
Of course :sweat_smile: